data/org.freedesktop.portal.Usb.xml

```diff
@@ -0,0 +1,228 @@
+        @options: Vardict with optional further information
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+
+        Acquires (i.e. open) the given device nodes. The are available until ReleaseDevice or until the dbus connection is terminated, in which case the file descriptor is closed.
+
+        Each element of the @devices array contains the device ID, and the
+        following keys:
+
+        * ``writable`` (``b``)
+
+          Whether the device will be opened in read-write or read-only mode.
+          Default: False
+
+        Supported keys in the @options vardict include:
+
+        * ``handle_token`` (``s``)
+
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation
+          for more information about the @handle.
+
+        The #org.freedesktop.portal.Request::Response signal is emitted without
+        any extra information.
+     -->
+    <method name="AcquireDevices">
+      <arg type="s" name="parent_window" direction="in"/>
+      <arg type="a(sa{sv})" name="devices" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        FinishAcquireDevices:
+        @options: Vardict with optional further information
+        @results: Array of device ids, and the result of the access
+        @finished: Whether all device results were reported
+
+        Retrieves the file descriptors of the devices requested during
+        org.freedesktop.portal.Usb.AcquireDevices().
+
+        This method can only be called once, and only after calling
+        org.freedesktop.portal.Usb.AcquireDevices().
+
+        Each element of the @devices_fds array contains the device ID, and the
+        following keys:
+
+        * ``success`` (``b``)
+
+          Whether the device access was successful or not.
+
+        * ``fd`` (``h``)
+
+          The file descriptor representing the device. The file
+          descriptor is meant to be passed on to the USB library.
+
+          Only present if this was a successful device access.
+
+        * ``error`` (``s``)
+
+          Error message describing why accessing the device was not
+          successful. Only present if this was an failed device access.
+
+        There are no supported keys in the @options vardict.
+     -->
+    <method name="FinishAcquireDevices">
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="a(sa{sv})" name="results" direction="out"/>
+      <arg type="b" name="finished" direction="out"/>
+    </method>
+
+    <!--
+        ReleaseDevices:
+        @devices: Array of device identifiers
+        @options: Vardict with optional further information
+
+        Releases (i.e. closes) previously acquired devices. The file
```

Comment:
This does not close the device. It only removes the record on the portal side and in the future hopefully revokes the device as well.

How about this?

> Releases (i.e. closes) previously acquired devices. The file descriptor returned by FinishAcquireDevices for each of the devices is closed.

> Releases previously acquired devices. The file descriptors of those devices might become unusable as a result of this.